dev/com.ibm.ws.concurrent.rx/bnd.bnd

```diff
@@ -0,0 +1,35 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: Completion Stage for Managed Executors
+Bundle-SymbolicName: com.ibm.ws.concurrent.rx
+Bundle-Description: Completion Stage for Managed Executors, version ${bVersion}
+
+WS-TraceGroup: concurrent
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+Export-Package: \
+  com.ibm.websphere.concurrent.rx
+
+instrument.classesExcludes: com/ibm/ws/concurrent/rx/resources/*.class
+  
+-buildpath: \
+	com.ibm.websphere.appserver.spi.logging;version=latest,\
+	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.ws.concurrent;version=latest,\
```

Comment:
do we need the dependency on `com.ibm.ws.concurrent;version=latest`?  Seems that just the API would be sufficient.

Reply:
We don't need it yet, but will, once we have updated the implementation to properly use context service/policy executor of the managed executor service instance.  This was preparing for the very near term future, so I will leave it here.